### ISSUE TITLE
about dice loss

### ISSUE THREAD

**Issue by kobe233333** [milesial/Pytorch-UNet#180]
I'm confused if we can use dice loss in training, and what should i do ?

**Comment by milesial** [comment]
DICE is just another metric that I used for the challenge because it makes sense for binary masks. You can try yourself to use it for training and backprop and see if it works for your dataset.

**Comment by kobe233333** [comment]
> DICE is just another metric that I used for the challenge because it makes sense for binary masks. You can try yourself to use it for training and backprop and see if it works for your dataset.

so when we do binary classification, if we can use the fuction in dice_loss.py to replace BCElogitloss in the training directly, or we have to write by ourselves?

**Comment by milesial** [comment]
Yes you can use it directly, the backward pass is already implemented (but not tested) https://github.com/milesial/Pytorch-UNet/blob/e5039519d57e39e7efb91183e29d39c049ba3100/dice_loss.py#L18
